omnibus/Gemfile

```diff
@@ -23,7 +13,13 @@ group :development do
   gem 'berkshelf', '>= 7.0'
 
   # Use Test Kitchen with Vagrant for converging the build environment
-  gem 'kitchen-vagrant', '~> 1.3.1'
-  gem 'test-kitchen',    '~> 1.21'
+  gem 'kitchen-vagrant', '>= 1.3.1'
+  gem 'test-kitchen', '>= 1.23'
   gem 'winrm-fs', '~> 1.0'
 end
+
+instance_eval(ENV['GEMFILE_MOD']) if ENV['GEMFILE_MOD']
```

Comment:
This seems odd to me. Is this supposed to be an arbitrary ruby string or a path to a file to eval in?

Reply:
@lamont-granquist how were you using this locally?